massa-execution-worker/src/controller.rs

```diff
@@ -169,6 +153,22 @@ impl ExecutionController for ExecutionControllerImpl {
         result
     }
 
+    /// Get a copy of a single datastore entry with its final and active values
+    ///
+    /// # Return value
+    /// * `Vec<(final_data_entry, active_data_entry)>`
+    fn get_final_and_active_data_entry(
+        &self,
+        input: Vec<(Address, Vec<u8>)>,
+    ) -> Vec<(Option<Vec<u8>>, Option<Vec<u8>>)> {
+        let lock = self.execution_state.read();
+        let mut result = Vec::with_capacity(input.len());
```

Comment:
swap the order of those 2 lines to minimize lock duration

Reply:
#3705

Reply:
thx